agent/stats/queue.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2014-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package stats
+
+import (
+	"fmt"
+	"math"
+	"sync"
+
+	"github.com/aws/amazon-ecs-agent/agent/acs/model/ecstcs"
+)
+
+const (
+	// CPUUsageUnit is the unit of CPU usage being reported for a container.
+	CPUUsageUnit = "Percent"
+
+	// MemoryUsageUnit is the unit of memory usage being reported for a container.
+	MemoryUsageUnit = "Megabytes"
+
+	// BytesInMB is the number of bytes in a MegaByte. Using MB as it is one of the
+	// units supported by Cloudwatch.
+	// refer http://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_MetricDatum.html
+	BytesInMB = 1000 * 1000
+)
+
+var cpuUsageUnit string
+var memoryUsageUnit string
+
+func init() {
```

Comment:
You don't need the init function; you can just say var cpuUsageUnit = CPUUsageUnit above. Could also throw a comment in that you can't use the consts because you need to take the address of them.